src/components/SlateEditor/plugins/details/onBackspace.ts

```diff
@@ -2,28 +2,30 @@
  * User pressed Delete in an editor:
  * Unwrap the blockquote if at the start of the inner block.
  */
-import { Point } from 'slate';
+import { Block, Document, Editor, Inline, Node, Point } from 'slate';
 import { findNodesByType } from '../../../../util/slateHelpers';
 
+type ParentNode = Document | Block | Inline | null;
+
 /*
-   editor.value.document.getOffset(node.key) is the offset of the current node where the cursor is. Blocknode is either the details or solution block. 
-   We also have to add the summaray text length to see if we should jump out of the block or delete text.
+   editor.value.document.getOffset(node.key) is the offset of the current node where the cursor is. Blocknode is the details block. 
+   We also have to add the summary text length to see if we should jump out of the block or delete text.
   */
-function handleBackspaceAndCheckOffsetinDetailsblock(
-  editor,
-  next,
-  start,
-  node,
-  blockNode,
-  summaryTextLength,
-) {
+const handleBackspaceAndCheckOffsetInDetailsblock = (
+  blockNode: Node,
+  editor: Editor,
+  next: () => void,
+  node: Node,
+  start: Point,
+  summaryTextLength: number,
+) => {
   if (
     editor.value.document.getOffset(node.key) ===
       editor.value.document.getOffset(blockNode.key) + summaryTextLength &&
     start.offset === 0
   ) {
     const firstPosition = Point.create({
-      key: editor.value.document.getPreviousBlock(blockNode.key).key,
+      key: editor.value.document.getPreviousBlock(blockNode.key)?.key,
```

Comment:
Her må vi bruke previousSibling for å finne korrekt blokk å hoppe til.
```suggestion
      key: editor.value.document.getPreviousSibling(blockNode.key)?.key,
```
Dette hopper til først på linja i forrige blokk. Kanskje det hadde vore bedre å finne offset til teksten inni blokken og så plassere markøren til slutt?

Reply:
Når jeg prøvde dette så hoppet cursoren ut av `details` direkte til `ekspanderende boks`, selvom det var tekst inni mellom som den burde hoppe til først.